### DISCUSSION TITLE
Older Retro Support

### DISCUSSION THREAD

**Discussion by Twitch0815** [gyf304/lpvpn#15] [General]
Thus far this has been working great just wanted to ask is it possible to add support for older OS's

add support for full network to network connection
example if I am connected on my windows 10 box  if any of the other boxs on my network could tunnel or see over the vpn
I could then have windows xp or windows 98 os on my network connect to local lan servers hosted by a player on the other side of the tunnel

Maybe as simple as a simple binary that tunnels additional nodes from older operating systems onto the same vpn

What is the max number of players supported or allowed to make a group in discord with connectivity

**Comment by gyf304** [comment]
You can try to bridge the VPN adapter and a USB adapter to your XP machine. (Or rather, I would just use an XP virtual machine for that). You can find plenty of tutorials online for this. (I did not try this setup I am not super sure if this will work).

Max number of player supported: currently hard-coded as 8, but can be up-ed in the future.

Edit: If you decide to bridge networks, please also manually remove the IP address on your Windows 10 VPN adapter, and use that IP address on your XP box to avoid IP conflict.
